core/common/src/test/java/alluxio/ConfigurationTestUtils.java

```diff
@@ -26,5 +31,58 @@ public static void resetConfiguration() {
     Configuration.init();
   }
 
+  /**
+   * Returns reasonable default configuration values for running integration tests.
+   *
+   * These defaults are mostly aimed at getting tests to run faster. Individual tests may override
+   * them to test specific functionality.
+   *
+   * @return the configuration
+   */
+  public static Map<PropertyKey, String> testConfigurationDefaults() {
```

Comment:
perhaps takes `workDirectory` and `hostname` so we could also set service addresses and tiered storage here too?

Reply:
done